packages/hardhat-core/src/internal/core/jsonrpc/types/output/transaction.ts

```diff
@@ -3,6 +3,13 @@ import * as t from "io-ts";
 import { nullable, optional } from "../../../../util/io-ts";
 import { rpcAddress, rpcData, rpcHash, rpcQuantity } from "../base-types";
 
+const rpcAccessListItem = t.type({
```

Comment:
These are shared by input and out types. Maybe we should move them `base-types.ts`? Not 100% convinced either.

Reply:
Oh, this is also repeated between transaction and call requests. Maybe it's not a base type, but could go at the same level.

Reply:
I thought about adding it to `base-types.ts`, but it felt very different to those types. I think a `access-list-types.ts` module at the same level as `base-types.ts` should work, wdyt?

Reply:
yeah, let's do that. I'll move them in my next pr